Gruntfile.js

```diff
@@ -351,12 +352,14 @@ var config = require('./core/server/config'),
                 },
 
                 prod: {
-                    'core/client': 'shell:ember:prod'
+                    'core/client': 'shell:ember:prod',
+                    'core/server/lib/members/static/auth': 'shell:preact:prod'
                 },
 
                 watch: {
                     projects: {
-                        'core/client': ['shell:ember:watch', '--live-reload-base-url="' + urlService.utils.getSubdir() + '/ghost/"']
+                        'core/client': ['shell:ember:watch', '--live-reload-base-url="' + urlService.utils.getSubdir() + '/ghost/"'],
+                        'core/server/lib/members/static/auth': ['shell:preact:dev']
```

Comment:
Runs watch and live reload for developing member auth views, probably one side-effect during local dev is it clears console once on start, this can be fixed in next iteration.